src/test/java/racingcar/view/InputViewTest.java

```diff
@@ -0,0 +1,137 @@
+
+        //when & then
+        Assertions.assertThrows(BlankInputException.class, InputView::inputCarNames);
+    }
+
+    @Test
+    @DisplayName("공백 입력 시 예외를 발생한다.")
+    void validateNonWhiteSpace() {
+        //given
+        String input = " ";
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+
+        //when & then
+        Assertions.assertThrows(BlankInputException.class, InputView::inputCarNames);
+    }
+
+    @ParameterizedTest
+    @DisplayName("입력받은 자동차 이름이 쉼표(,)로 구분되지 않으면 예외가 발생한다.")
+    @ValueSource(strings = {"a.b.c,", "a;b;c", "a/b/c", "a-b-c", "a+b+c"})
+    void validateCommaDelimitedNames(String input) {
+        //given
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+
+        //when & then
+        Assertions.assertThrows(CarNameFormatException.class, InputView::inputCarNames);
+    }
+
+    @ParameterizedTest
+    @DisplayName("입력받은 자동차 이름들의 사이가 아닌 바깥에 쉼표가 있는 경우 예외가 발생한다.")
+    @ValueSource(strings = {"a,b,c,", ",a,b,c", ",a,b,c,", ",abc", "abc,"})
+    void validateCommaBetweenCarNames(String input) {
+        //given
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+
+        //when & then
+        Assertions.assertThrows(CarNameFormatException.class, InputView::inputCarNames);
+    }
+
+    @ParameterizedTest
+    @DisplayName("자동차 이름에 특수문자가 올 경우 예외를 발생한다.")
+    @ValueSource(strings = {"a,b,.", "a,;,c", "@,@,@", "*,*,*"})
+    void validateNonSpecialCharacter(String input) {
+        //given
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+
+        //when & then
+        Assertions.assertThrows(CarNameFormatException.class, InputView::inputCarNames);
+    }
+
+    @ParameterizedTest
+    @DisplayName("자동차 이름에 알파벳, 숫자, 한글을 입력할 수 있다.")
+    @ValueSource(strings = {"a,b,c", "1,2,3", "가,나,다"})
+    void validateAlphabetAndNumberAndKorean(String input) {
+        //given
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+
+        //when & then
+        org.assertj.core.api.Assertions.assertThatCode(InputView::inputCarNames)
+                .doesNotThrowAnyException();
+    }
+
+    @ParameterizedTest
+    @DisplayName("입력받은 경주 횟수가 숫자가 아닌 경우, 예외를 발생한다.")
+    @ValueSource(strings = {"가", ".", "%", "+", "^"})
+    void validateOnlyNumberRound(String input) {
+        //given
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+
+        //when & then
+        Assertions.assertThrows(RoundNumberFormatException.class, InputView::inputRoundNumber);
+    }
+
+    @ParameterizedTest
+    @DisplayName("경주 횟수에 숫자를 입력한다.")
+    @ValueSource(strings = {"5", "05", "005", "0005", "000005", "10", "20"})
+    void validateInputNumber(String input) {
+        //given
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+
+        //when & then
+        org.assertj.core.api.Assertions.assertThatCode(InputView::inputRoundNumber)
+                .doesNotThrowAnyException();
```

Comment:
assertThatCode()를 static import하면 아래와 같이 쓸 수 있을 것 같아요!

```suggestion
        assertThatCode(InputView::inputRoundNumber).doesNotThrowAnyException();
```

Reply:
구글 자바 스타일 컨벤션에서 "static 또는 기타 와일드 카드 import 는 사용하지 않습니다 ." 라는 문구를 보고 사용하지 않았습니다! 이유를 찾아보니 해당 코드가 어떤 클래스로부터 온 것인지 확인하기 힘들다고 하더라구요. 이 부분에 대해서 어떻게 생각하시는지 궁금합니다!

Reply:
`assertThatCode()`의 경우에는 static import를 하더라도 충분히 `org.assertj.core.api.Assertions`에서 가져왔다는 것을 파악할 수 있을 것 같아요! 그래서 제안 드렸습니다